quiz.js

```diff
@@ -1,168 +1,137 @@
 $(document).ready(function () {
-
     //DOM Variables
-    const scoreEl = $("#score")
-    const categoryEl = $("#category");
-    const questionEl = $("#question");
-    const answerZoneEl = $("#answer-zone");
-    const rightWrongEl = $("#right-wrong");
+    const scoreEl          = $("#score")
+    const categoryEl       = $("#category");
+    const questionEl       = $("#question");
+    const answerZoneEl     = $("#answer-zone");
+    const rightWrongEl     = $("#right-wrong");
     const questionNumberEl = $("#question-number");
 
     //initial variables
-    let questions = [];
-    let score = 0;
+    let questions     = [];
+    let score         = 0;
     let questionIndex = 0;
     let correctAnswer;
 
     //audio variables
-
-    //crowd cheer sound
     const cheer = new Audio('./Assets/audio/bbc_crowds--ch_07043047.mp3');
-    //crow jeer sound
-    const jeer = new Audio("./Assets/audio/bbc_crowd-reac_07018034.mp3")
-    // MF DOOM backtrack
-    const DOOM = new Audio("./Assets/audio/Metal Fingers Special Herbs (Volume 1&2) [FULL ALBUM].mp3")
+    const jeer  = new Audio("./Assets/audio/bbc_crowd-reac_07018034.mp3")
+    const DOOM  = new Audio("./Assets/audio/Metal Fingers Special Herbs (Volume 1&2) [FULL ALBUM].mp3")
```

Comment:
I don't particularly see a need for comments that are explained by the code they're referring to. Maybe that's just with more experience 
```
//crowd cheer sound // <-- this comment is a bit redundant since the 
                    //     following line says "here's an audio object named 'cheer'"

    const cheer = new Audio('./Assets/audio/bbc_crowds--ch_07043047.mp3');
```